docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -6337,6 +6337,300 @@ UPDATE persnl.job
 SET jobdesc = RIGHT (jobdesc, 12);
 ```
 
+<<<
+[[rollup_function]]
```

Comment:
I think this is not a function but a clause, so I suggest move this chapter into chapter 6?
I am not sure about this, @DaveBirdsall what do you think here?